master/internal/command/notebook_manager.go

```diff
@@ -43,9 +43,14 @@ func (n *notebookManager) Receive(ctx *actor.Context) error {
 		}
 		for _, notebook := range ctx.AskAll(&notebookv1.Notebook{}, ctx.Children()...).GetAll() {
 			typed := notebook.(*notebookv1.Notebook)
-			if (len(users) == 0 && len(userIds) == 0) || users[typed.Username] || userIds[typed.UserId] {
-				resp.Notebooks = append(resp.Notebooks, typed)
+			if !((len(users) == 0 && len(userIds) == 0) || users[typed.Username] || userIds[typed.UserId]) {
```

Comment:
there is a better way of articulating this probably

Reply:
I've given this some thought and I can't come up with anything better. If we want to make it super explicit, maybe add a helper function that just evaluates this? But even then, I think just reading the code to understand takes a couple minutes at most and is sufficient.